iceberg/iceberg-handler/src/test/results/positive/iceberg_atomic_merge_update.q.out

```diff
@@ -234,8 +234,8 @@ POSTHOOK: type: QUERY
 POSTHOOK: Input: default@calls_v2
 POSTHOOK: Output: hdfs://### HDFS PATH ###
 append	10	NULL
+delete	NULL	4
```

Comment:
is that expected?

Reply:
The operation has been changed from an append to a metadata delete which is why it has changed.

Reply:
Delete query is present - [https://github.com/apache/hive/blob/b6847ed38b7d32586ab22e224904867f159b510e/icebe[…]handler/src/test/queries/positive/iceberg_atomic_merge_update.q](https://github.com/apache/hive/blob/b6847ed38b7d32586ab22e224904867f159b510e/iceberg/iceberg-handler/src/test/queries/positive/iceberg_atomic_merge_update.q#L92)
In the iceberg_atomic_merge_update.q